### ISSUE TITLE
Getting older versions of packages

### ISSUE THREAD

**Issue by arunoda** [rissem/meteorpad#16]
For what ever the reason, it's not getting the latest version of the packages.

**Comment by rissem** [comment]
Right now the package catalog is updated on release. It'll be fixed to update continuously soon. I just released, so it should have the latest now.

**Comment by arunoda** [comment]
Thanks Mike.
I'll try it now.

BTW: Isn't it loading packages from the official repo? Are you caching it in some middle layer?

**Comment by liron00** [comment]
I published a new package to Atmosphere called "quixey:react" and I'm hoping to demo it with MeteorPad, but I can't get it to appear in the dropdown.

**Comment by rissem** [comment]
@arunoda yeah right now there's a caching layer for faster searching. I'm planning to release something better soon.

@liron00 you should be able see your package now.

**Comment by liron00** [comment]
Awesome thanks!

--
*Liron Shapira*, Chief Science Officer
US: +1 262.528.6671 | 303 Bryant St, Mountain View, CA 94041
Israel: +972 54.457.5776 | Azrieli Round Tower Fl 29, 132 Menachem Begin
Rd, Tel Aviv, Israel 67021
<http://i.imgur.com/dpMvVRI.png>

**Comment by dgreensp** [comment]
Adding `peerlibrary:blaze-components` to a new app on MeteorPad gives you version 0.1.0, whereas adding it on the command-line with Meteor correctly gives you the latest version 0.7.0.  This was reported in a Meteor issue thread, but it sounds like a MeteorPad bug.
